modules/templates/helper.go

```diff
@@ -285,6 +285,13 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return false
 		},
+		"octicon": func(mega bool, icon string) template.HTML {
```

Comment:
I know this request will change a lot of your work below, however is it possible to use icon as the first argument in this function? The reason is that it is the most important argument, and that mega is a modifier.

This isn't a blocker for me though.

Reply:
Thankfully this should be specific enough in templates to mass search-replace.

Reply:
Done